packages/react-composites/tests/browser/chat/FileSharing.test.ts

```diff
@@ -233,7 +232,7 @@ test.describe('Filesharing Global Errorbar', async () => {
     await stubMessageTimestamps(page);
 
     await page.locator(dataUiId('file-download-card-download-icon')).click();
-    await clickOutsideOfPage(page);
+    await page.addStyleTag({ content: '.ui-chat__message__actions {display: none}' });
```

Comment:
Instead of adding custom logic here. Can we try using `stableScreenshot` function here as advised by the `clickOutsideOfPage` deprecation docs

Reply:
done